### PR TITLE
Prepare the repo for codecov.io representation

### PR DESCRIPTION
**Pull request by Dee-6777**
### What this PR does / Why we need it?
Prepare `kite` for codecov.io

### Which Jira issue(s) does this PR fix?
https://issues.redhat.com/browse/OSD-15457

### CODE REVIEW

**Comment by Dee-6777** [comment]
Need approval of the PR : [openshift/release/pull/37072] to retest this PR.

File: Makefile [hunk 1]
```diff
@@ -34,25 +35,12 @@ tools:
...
-	set -e ;\
-	REQ_COV=15.5 ;\
-	TEST_COV=$$(go tool cover -func coverage.out | grep -F total | awk '{print substr($$3, 1, length($$3)-1)}' ) ;\
-	if (( $$(echo "$$REQ_COV > $$TEST_COV" |bc -l) )); then echo "Error: Code Coverage is less"; exit 1 ;\
-	else echo "Code Coverage Test Passed"; exit 0; fi ;\
-	}
-	@rm -rf coverage.out
+.PHONY: coverage
+coverage:
+	chmod 755 hack/codecov.sh
```
Comment (supreeth7) [inline Makefile:42]: This shouldn't be done in the Makefile. Please perform the locally and change the permissions of `hack/codecov.sh` file to be executable by others.

**Review by supreeth7 — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (Dee-6777) [reply Makefile]: Yes I removed that. Thanks

**Pull request merged**